README.md

```diff
@@ -49,3 +50,19 @@ When a commit is pushed into `X.Y`:
 ## License
 
 See LICENSE files in individual subdirectories
+
+## The process (TODO)
```

Comment:
Rather than keeping it in the README, perhaps it's an idea to create issues for each item. Then you can add a [milestone](https://github.com/theforeman/foreman-documentation/milestones) and link all issues. Here you can link to the milestone.

Reply:
@melcorr thoughts on how you want to do this? I think it's a good idea.

Reply:
Yeah I think it is a great idea. The list is a bit of an assortment, like a mind dump from the first pass through this project, which is exactly what I think it is. I think we should remove it in a further PR and define short-term goals and long term goals. I'll open an issue where maybe we can make a list of other issues to create.